docs/_snippets/button-story.md

```diff
@@ -449,46 +449,47 @@ export const Primary: Story = {
 };
 ```
 
-```js filename="Button.stories.js" renderer="svelte" language="js"
-import Button from './Button.svelte';
+```svelte filename="Button.stories.svelte" renderer="svelte" language="js" tabTitle="Svelte CSF"
```

Comment:
AFAIK these snippets aren't included in the docs, right? because `IfRenderer` only targets React and Solid.

Reply:
Yes @JReinhold, I took a look and it seems that this was a bit of an issue with the migration. You can remove the changes here

Reply:
let's just keep it, doesn't hurt to be more correct I guess.

Reply:
Sounds good to me